test/functional/rpc_users.py

```diff
@@ -84,6 +86,39 @@ def test_auth(self, node, user, password):
         self.log.info('Wrong...')
         assert_equal(401, call_with_auth(node, user + 'wrong', password + 'wrong').status)
 
+    def test_rpccookieperms(self):
+        p = {"owner": 0o600, "group": 0o640, "all": 0o644}
+
+        if os.name == 'nt':
```

Comment:
In 31cc8bc305103baf2c9979d86ef1b43dec5628cb "test: add rpccookieperms test"

Since #29034, the preferred convention is to use `platform.system()` rather than `os.name`.

Reply:
Thanks, updated in 49d5bfdd7e7